pages/deployments/oidc/oidc_with_aws.md

```diff
@@ -0,0 +1,168 @@
+---
+keywords: oidc, authentication, IAM, roles, AWS
+---
+
+# OIDC with AWS
+
+The [Buildkite Agent's oidc command](/docs/agent/v3/cli-oidc) allows you to request an OIDC token representing the current job. These tokens can be exchanged on federated systems like AWS for an identity and access management (IAM) role with AWS-scoped permissions.
```

Comment:
```suggestion
The [Buildkite Agent's oidc command](/docs/agent/v3/cli-oidc) allows you to request an OIDC token representing the current job. These tokens can be exchanged on federated systems like AWS for an Identity and Access Management (IAM) role with AWS-scoped permissions.
```

Reply:
Looking at the AWS docs, it looks like IAM is a product rather than feature name (at least that's how Amazon is conveying this in their docs, even though Amazon don't appear to offer clear style guidance on distinguishing the two). Therefore, let's follow suit in our docs.